lib/sober_swag/reporting/output/via_map.rb

```diff
@@ -0,0 +1,43 @@
+module SoberSwag
+  module Reporting
+    module Output
+      ##
+      # Apply a mapping function before calling
+      # a base output.
+      class ViaMap < Base
```

Comment:
Why is this class `ViaMap` named differently than the equivalent class `Mapped` in the input section?

Reply:
5ca038f